jdaviz/configs/imviz/tests/test_regions.py

```diff
@@ -70,29 +70,29 @@ def test_regions_invalid(self):
 
         # Make sure nothing is actually loaded
         self.verify_region_loaded('MaskedSubset 1', count=0)
-        assert self.imviz.get_interactive_regions() == {}
+        assert self.imviz.plugins['Subset Tools']._obj.get_subsets_as_regions() == {}

```

Comment:
don't need `._obj` if `get_subsets_as_regions` is exposed (which it currently is in this PR).